srcpkgs/sddm/template

```diff
@@ -30,6 +30,7 @@ conf_files="
 make_dirs="/var/lib/sddm 0755 sddm sddm"
 
 post_install() {
-	sed -i 's,systemd,ck_connector,g' ${DESTDIR}/etc/pam.d/sddm-greeter
+	vsed -i 's,systemd,ck_connector,g' ${DESTDIR}/etc/pam.d/sddm-greeter
+	vsed -i '/tally/d' ${DESTDIR}/etc/pam.d/sddm-autologin
```

Comment:
please provide additional information/context to the commit why you remove it

Reply:
Should be done